homeassistant/components/config/entity_registry.py

```diff
@@ -36,14 +36,18 @@ def _async_clear_list_entities_cache(event: Event) -> None:
         {vol.Required("type"): "config/entity_registry/list"}
     )
     @callback
-    def websocket_list_entities(hass, connection, msg):
+    def websocket_list_entities(
+        hass: HomeAssistant,
+        connection: websocket_api.ActiveConnection,
+        msg: dict[str, Any],
+    ) -> None:
         """Handle list registry entries command."""
         nonlocal cached_list_entities
         if not cached_list_entities:
             registry = er.async_get(hass)
             cached_list_entities = message_to_json(
                 websocket_api.result_message(
-                    IDEN_TEMPLATE,
+                    IDEN_TEMPLATE,  # type: ignore[arg-type]
```

Comment:
I added a `type ignore` here because it expects an `int`.
This is a special case, and we don't want all websocket methods to start sending strings, so I didn't chose not to adjust the `result_message` definition.